util/src/main/java/io/kubernetes/client/util/leaderelection/Lock.java

```diff
@@ -0,0 +1,16 @@
+package io.kubernetes.client.util.leaderelection;
+
+public interface Lock {
```

Comment:
Since this is an interface, I think it needs more documentation.

Reply:
+1